examples/clean-architecture/use-cases/notes/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Since this is just a simple CRUD, I'm only calling the underlying repository methods.
+ * In a real world application, in the same use-case method, you will insert the entire business logic.
+ */
+
+function NotesUseCase(notesRepository) {
```

Comment:
Can you call that NotesService or NotesBusinessService and also the paramter in the controller? 
just for maintain the classic architecture naming convention Controller-Service-Repository.